pkg/generators/merge.go

```diff
@@ -0,0 +1,192 @@
+package generators
```

Comment:
Missing godocs in this file

Reply:
Aw c'mon docs are for nerds.

jk, will add. :-)